fenix/app/src/main/java/org/mozilla/fenix/compose/Menu.kt

```diff
@@ -62,14 +67,37 @@ private fun Menu(
                 .background(color = FirefoxTheme.colors.layer2)
                 .then(modifier),
         ) {
+            val atLeastOneItemChecked = menuItems.any { it.isChecked }
+
             for (item in menuItems) {
                 DropdownMenuItem(
-                    modifier = Modifier.testTag(item.testTag),
+                    modifier = Modifier
+                        .testTag(item.testTag)
+                        .align(alignment = Alignment.CenterHorizontally),
                     onClick = {
                         onDismissRequest()
                         item.onClick()
                     },
                 ) {
+                    if (atLeastOneItemChecked) {
+                        if (item.isChecked) {
+                            Icon(
+                                painter = painterResource(id = R.drawable.mozac_ic_checkmark_24),
+                                modifier = Modifier
+                                    .padding(end = 12.dp)
+                                    .size(24.dp),
+                                contentDescription = null,
+                                tint = FirefoxTheme.colors.iconPrimary,
+                            )
+                        } else {
+                            Spacer(
+                                modifier = Modifier
+                                    .padding(end = 12.dp)
```

Comment:
Could we incorporate this into the `size()`?

Reply:
Done